handlers/handler.go

```diff
@@ -1,23 +1,15 @@
 package handlers
 
 import (
-	"crypto/rsa"
 	"errors"
-	"fmt"
-	"net/http"
-	"strings"
-
-	rice "github.com/GeertJohan/go.rice"
```

Comment:
Do you use the `go import` tool? https://godoc.org/golang.org/x/tools/cmd/goimports
This automatically formats the order of imports and also adds/removes imports. Actually the defacto standard.